products/sle/main.pm

```diff
@@ -1581,9 +1567,7 @@ else {
             loadtest "rt/boot_rt_kernel";
         }
         else {
-            if (get_var('S390_ZKVM')) {
-                loadtest 'installation/bootloader_zkvm';
-            }
+            load_bootloader_s390x();
```

Comment:
it's not just refactoring , it's change in behavior .
before you trigger something only if it is S390 with ZKVM backend 
after this change you will trigger bootloader_zkvm for ZKVM AND bootloader_s390 in other case

Reply:
There can be some corner cases, but in general it will be same behavior. For example this S390_ZKVM variable is set to 1 if backend is scirt and arch is s390x which corresponds to the extracted logic. @Soulofdestiny, please take a look

Reply:
generally agreed with anton here, but since we don't have any test (and it doesn't make any sense to add one) which is z/VM and uses BOOT_HDD_IMAGE, it should be fine

Reply:
I was looking into s390x tests in o.s.d (Functional, Migration and Userspace Regression) and there is no build which would use `bootloader_s390` and `boot_to_desktop`. All users of `boot_to_desktop` are these with `bootloader_zkvm`.